helpers/nasa_helpers.rb

```diff
@@ -0,0 +1,30 @@
+module NASA
+  module Helpers
+    def valid(plateau_coordinates, rover_parameters)
```

Comment:
Validation Helper to make sure the parameters passed are exactly what's expected